editor/src/core/model/element-metadata-utils.ts

```diff
@@ -2080,9 +2073,11 @@ function fillSpyOnlyMetadata(
 
     workingElements[pathStr] = {
       ...spyElem,
+      ...domElem,
```

Comment:
this feels like a massive unrelated, change to me that has barely anything to do with globalContentBoxForChildren, and _more important_ than the content box. if we need this change, I think this deserves a full explanation of the original behavior, how it was bad, the new behavior, how it is good, and also tests that demonstrate both

Reply:
reverted in https://github.com/concrete-utopia/utopia/pull/3483/commits/9040fa0cd45ae9501182281f97398f6c3b373c68